client/src/modules/accounts/accounts.service.js

```diff
@@ -2,17 +2,17 @@ angular.module('bhima.services')
   .service('AccountService', AccountService);
 
 AccountService.$inject = [
-  'PrototypeApiService', 'bhConstants',
+  'PrototypeApiService', 'bhConstants', 'util',
 ];
 
 /**
  * Account Service
  *
  * A service wrapper for the /accounts HTTP endpoint.
  */
-function AccountService(Api, bhConstants) {
-  var baseUrl = '/accounts/';
-  var service = new Api(baseUrl);
+function AccountService(Api, bhConstants, util) {
```

Comment:
There isn't any need to inject `util` here.  It's already available on `service` as `service.util`.